src/pp.js

```diff
@@ -78,6 +78,13 @@ function iterDoc(topDoc, func) {
         for (var i = doc.parts.length - 1; i >= 0; i--) {
           docs.push(doc.parts[i]);
         }
+      } else if (doc.type === "if-break") {
+        if (doc.breakContents !== undefined) {
```

Comment:
So yeah I think this is fine as I think being more explicit (an empty string) has a nice symmetry to it